tests/framework/extensions/corral/cleanup.go

```diff
@@ -0,0 +1,15 @@
+package corral
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/corral"
+)
+
+func CleanUpCorrals(corralList map[string]string) error {
```

Comment:
Why not just list all the corrals and delete them, instead of taking map argument